src/app/layout/authenticationPage/loginPage/LoginPage.tsx

```diff
@@ -0,0 +1,112 @@
+import { loginUser } from '@/api/authentication';
+import useAuth from '@/app/common/hooks/useAuth';
+import Button from '@/app/components/buttons/Button/Button';
+import { LoginForm, loginValidationSchema } from '@/types/entities/Authentication';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { IconButton, InputAdornment, TextField } from '@mui/material';
+import { useMemo, useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import './LoginPage.scss';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+
+interface LoginPageProps {
+  setIsLoginForm: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const LoginPage = ({ setIsLoginForm }: LoginPageProps) => {
+  const { setIsLoggedIn } = useAuth();
+
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+
+  const currentYear: number = useMemo(() => {
+    return new Date().getFullYear();
+  }, []);
+
+  const onLogIn = async (data: LoginForm) => {
+    loginUser(data.username, data.password, data.stayLoggedIn)
+      .then(() => {
+        setIsLoggedIn(true);
+        reset();
+      })
+      .catch(() => alert('an error occured'));
```

Comment:
Just FYI, I remeber I created for one of my tasks `src\app\components\alerts\AppAlert.tsx` - popup to display messages, warnings, errors, etc

Reply:
Done, added previously that task to jira